app/components/addAppointmentButton.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { TouchableOpacity, StyleSheet, Image } from 'react-native';
+import Colors from '../utils/Colors';
+
+const calenderAddIcon = require('../../assets/calendar_add.png');
+
+interface AddButtonProps {
+    onPress: () => void;
+}
+
+const AddButton: React.FC<AddButtonProps> = ({ onPress }) => {
+    return (
+        <TouchableOpacity style={styles.button} onPress={onPress}>
+            <Image 
+                source={calenderAddIcon}
+                style={{ width: 25, height: 25 }}
+                resizeMode='contain'
+            />
+        </TouchableOpacity>
+    );
+};
+
```

Comment:
Since the button is only used once in the appointmentPage and the implementation is really small, could you move it just inside the appointments.tsx component?

Reply:
done :)